ga4gh/frontend.py

```diff
@@ -169,6 +173,7 @@ def searchVariantSets():
     return handleFlaskRequest(flask.request, app.backend.searchVariantSets)
 
 
-@app.route('/variants/search', methods=['POST', 'OPTIONS'])
+@app.route('/<version>/variants/search', methods=['POST', 'OPTIONS'])
+@versioningManager.policy('V_0.5.1_RELEASE')
```

Comment:
To make it a unique string, for easier grepping through the code.

A case could be made the other way, for consistency.

I suppose I prefer it the way I did it, to avoid entangling identifiers, but I don't have strong feelings on this decision.